src/Controller/User/UserController.php

```diff
@@ -70,4 +79,27 @@ public function userAccount(ChartBuilderInterface $chartBuilder): Response
             'chart' => $chart ?? null,
         ]);
     }
+
+    #[Route(path: '/user/participation', name: 'user_participation')]
+    public function test(Request $request)
+    {
+        $participation = new Participation($this->getUser());
+        $form = $this->createForm(ParticipationType::class, $participation);
+
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
+            $this->em->persist($participation);
+            $this->em->flush();
+
+            return $this->redirectToRoute('user_participation');
```

Comment:
you redirect the user to the current route. No need to do that, just continue the execution of the action